independent-projects/resteasy-reactive/common/processor/src/main/java/org/jboss/resteasy/reactive/common/processor/EndpointIndexer.java

```diff
@@ -1380,8 +1380,11 @@ protected void handleSortedSetParam(Map<String, String> existingConverters, Stri
             boolean hasRuntimeConverters, PARAM builder, String elementType) {
     }
 
-    protected void handleOptionalParam(Map<String, String> existingConverters, String errorLocation,
-            boolean hasRuntimeConverters, PARAM builder, String elementType, String genericElementType) {
+    protected void handleOptionalParam(Map<String, String> existingConverters,
+            Map<DotName, AnnotationInstance> parameterAnnotations,
+            String errorLocation,
+            boolean hasRuntimeConverters, PARAM builder, String elementType, String genericElementType,
+            MethodInfo currentMethodInfo) {
```

Comment:
Is it already supported by other containers? List, Set...

Reply:
No it does not.

Reply:
Should it? :)

Reply:
Likely yes, but that requires a different fix, which is probably best done in a follow up

Reply:
BTW, It's an honest question, I don't know if it makes sense.

Reply:
Yeah, it does :). I was iniatially sceptical, but came to the conclusion that does make sense.